test/integrations/destinations/the_trade_desk/delivery/other.ts

```diff
@@ -0,0 +1,78 @@
+import { ProxyV1TestData } from '../../../testTypes';
+import { generateProxyV1Payload, generateMetadata } from '../../../testUtils';
+import {
+  destType,
+  advertiserId,
+  dataProviderId,
+  segmentName,
+  proxyV1PlatformErrorStatTags,
+  firstPartyDataEndpoint,
+} from '../common';
+import { envMock } from '../mocks';
+
+envMock();
+
+const validRequestPayload1 = {
+  AdvertiserId: advertiserId,
+  DataProviderId: dataProviderId,
+  Items: [
+    {
+      DAID: 'test-daid-1',
+      Data: [
+        {
+          Name: segmentName,
+          TTLInMinutes: 43200,
+        },
+      ],
+    },
+  ],
+};
+
+const metadataArray = [generateMetadata(1)];
+
+export const otherProxyV1: ProxyV1TestData[] = [
```

Comment:
Please add the network failure scenarios apart from these in `other.ts` check braze/ cm360 for reference

Reply:
added